decidim-core/lib/tasks/decidim_metrics_tasks.rake

```diff
@@ -43,15 +43,16 @@ namespace :decidim do
         raise ArgumentError if day.blank?
 
         (Date.parse(day)..Time.zone.today).each do |d|
```

Comment:
instead of `Time.zone.today` why don't use `Date.current`?

Reply:
I don't know, why change it? I guess both will always return the same no? readability?

Reply:
Yes I think so also, but for readability and as it concerns dates `Date.current` seems more appropiate to me :)